shared-helpers/src/locales/tl.json

```diff
@@ -181,21 +181,18 @@
   "application.financial.vouchers.title": "Ikaw ba o sinuman sa iyong sambahayan ay tumatanggap ng Seksyon 8 o Housing Authority na inisyu ng mga voucher, kabilang ang mga voucher ng Veterans Affairs Supportive Housing (VASH), o tulong sa pag-upa mula sa ibang mga mapagkukunan?",
   "application.form.general.saveAndFinishLater": "I-save at tapusin pagkatapos",
   "application.form.general.saveAndReturn": "I-save at ibalik para repasuhin",
-  "application.form.options.relationship.aunt": "Tiyahin",
+  "application.form.options.relationship.auntUncle": "Tita/Tito",
   "application.form.options.relationship.child": "Anak",
   "application.form.options.relationship.cousin": "Pinsan",
   "application.form.options.relationship.friend": "Kaibigan",
-  "application.form.options.relationship.grandparent": "Lolo o Lola",
-  "application.form.options.relationship.greatGrandparent": "Lolo o Lola sa Tuhod",
-  "application.form.options.relationship.inLaw": "Biyenan o Bayaw o Manugang",
-  "application.form.options.relationship.nephew": "Pamangkin na lalaki",
-  "application.form.options.relationship.niece": "Pamangkin na babae",
+  "application.form.options.relationship.grandparentGreatGrandparent": "Lolo o Lola/Lolo o Lola sa Tuhod",
+  "application.form.options.relationship.nephewNiece": "Pamangkin na lalaki/Pamangkin na babae",
+  "application.form.options.relationship.liveInAide": "Live-in Aide",
   "application.form.options.relationship.other": "Iba pa",
   "application.form.options.relationship.parent": "Magulang",
-  "application.form.options.relationship.registeredDomesticPartner": "Nakarehistrong kinakasama",
-  "application.form.options.relationship.sibling": "Kapatid",
-  "application.form.options.relationship.spouse": "Asawa",
-  "application.form.options.relationship.uncle": "Tiyuhin",
+  "application.form.options.relationship.girlfriendBoyfriend": "Kasintahan",
```

Comment:
Girlfriend and Boyfriend both translate to Kasintahan so I chose to include it just once rather than Kasintahan/Kasintahan